umich_api/api_utils.py

```diff
@@ -51,15 +54,19 @@ def __init__(self, base_url, client_id, client_secret, api_json = None):
         with open(api_json, encoding='utf-8') as api_file:
             apis = json.loads(api_file.read())
 
```

Comment:
Here a check should be taken into account that file don't have empty string

Reply:
This will load the default from the package if the api_file is empty. I guess if someone puts an empty json in there then this won't work or throw an exception from the json parsing. I don't think this is something this has to be caught here. 

For these cases we'd have to print a message and re-raise the exception or just let the exception be raised and let the application that called it fix it. I think that's fine for this?

Reply:
Ya... raising an exception should be fine. Logging a msg would be good as a courtesy.


```
msg = 'api.json file is empty, please add some api scope'
 raise Exception(msg)
```